ajax/libs/clarity-icons/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "clarity-icons",
+  "version": "0.9.16",
+  "filename": "clarity-icons.min.js",
+  "description": "Custom Element Icons for Clarity",
+  "homepage": "https://vmware.github.io/clarity/",
+  "repository": {
+    "type": "git",
+    "url": "ssh://git@git.eng.vmware.com/clarity.git"
```

Comment:
I'll update to `git+https://github.com/vmware/clarity.git`
Although https://github.com/vmware/clarity/blob/master/build/npm/clarity-icons.json#L10, the link can't work.